### DISCUSSION TITLE
Code not working

### DISCUSSION THREAD

**Discussion by wifi75** [maxill1/ialarm-mqtt#39] [Q&A]
Hello I have add code on my configuration below hadiscovery.
but not working why?
![image](https://user-images.githubusercontent.com/31816231/202390595-dce5f444-9799-418b-bd8f-669ae58b5f1a.png)
![image](https://user-images.githubusercontent.com/31816231/202390648-359d52ae-e3f2-4328-927d-70610b25f13a.png)

**Comment by Nik71git** [comment]
il codice deve avere solo 4 cifre

**Comment by wifi75** [comment]
ho provato con 4 ma non funziona ugualmente ho provato a copiare la voce code sia sotto hadiscovery che sotto server, ma non appare il tastierino numerico...

**Comment by maxill1** [comment]
confermo che da addon non funziona, mentre da docker/npx/npm si. Devo verificare come mai l'addon non passa la configurazione a ialarm-mqtt.

**Reply by Nik71git** [reply]
ok, grazie. se serve fare da debugger per l'addon di HAOS sono a disposizione.

**Reply by maxill1** [reply]
dovrebbe essere risolta sull'addon ialarm-mqtt-dev

**Reply by Eckart0** [reply]
Ciao, sto usando l'AddOn con un impianto Meian-Focus FC7668Pro. Tutto funziona come dovrebbe. Tranne che: non riesco a far sì che alarm_control_panel richieda l'inserimento di un codice. Ho configurato l'AddOn sia direttamente nella pagina di configurazione dell'AddOn, inserendo il "Codice: 1234", sia nel file configuration.yaml sotto "mqtt:". Non funziona nulla, qualsiasi ladro intraprendente può disattivare l'impianto senza inserire il codice. C'è qualche impostazione che mi è sfuggita? Cordiali saluti e grazie mille per questo AddOn! Eckart

**Reply by Eckart0** [reply]
Hello, I am using the AddOn with a Meian-Focus FC7668Pro system. Everything works as it should. Except that: I cannot get alarm_control_panel to request a code to be entered. I have configured the AddOn both directly on the AddOn configuration page, entering "Code: 1234", and in the configuration.yaml file under "mqtt:". Nothing works, any resourceful burglar can deactivate the system without entering the code. Is there some setting I have missed? Kind regards and many thanks for this AddOn! Eckart
